### PR TITLE
Update docs for link parameter formatting

### PR DESCRIPTION
**Pull request by drewcorlin1**
## Which problem is this PR solving?
Documentation associated with https://github.com/jaegertracing/jaeger-ui/pull/2501

### CODE REVIEW

File: content/docs/next-release-v2/frontend-ui.md [hunk 1]
```diff
@@ -146,6 +151,13 @@ For traces, the supported template fields are: `duration`, `endTime`, `startTime
 
 Further, the trace template fields are available for substitution in process/logs/tags type when the trace template fields are prefixed with `trace.`. For example: `trace.traceID`, `trace.startTime`.
 
+#### Formatting
+
+In addition to interpolating fields into links, formatting functions can be used. The syntax is `#{field | function}` (eg `#{endTime | epoch_micros_to_date_iso}'`). The available formatting functions and a description of their behavior are:
+| Function | Description | Example |
+| -------- | ----------- | ------- |
+| `epoch_micros_to_date_iso` | Format a date in microseconds since epoch to an ISO date time | `#{endTime \| epoch_micros_to_date_iso}` |
```
Comment (drewcorlin1) [inline content/docs/next-release-v2/frontend-ui.md:159]: I can imagine additions to this list. A few that are on my mind to being
* left padding traceId with "0"s, for example in jaeger I have a trace ID "42c7668cd8d887da" which shows up in my logs as "000000000000000042c7668cd8d887da" with the OTel pino instrumentation
* Allowing time offsets to be passed to `epoch_micros_to_date_iso` to allow for time skew (eg something like `#{endTime | epoch_micros_to_date_iso 10}` which would add 10 microseconds to the end of the time before formatting it, syntax TBD of course)

**Review by yurishkuro — APPROVED** [review]

**Comment by yurishkuro** [comment]
please amend the commit with sing-off `git commit -s --amend`

**Comment by yurishkuro** [comment]
Please check linters

File: content/docs/next-release-v2/frontend-ui.md [hunk 2]
```diff
@@ -66,6 +66,11 @@ An example configuration file (see [complete schema here](https://github.com/jae
     "key": "uniqueId",
     "url": "https://mykibana.com/uniqueId=#{uniqueId}&traceId=#{trace.traceID}",
     "text": "Redirect to kibana to view log"
+  },
+  {
+    "type": "traces",
+    "url": "https://mykibana.com/_dashboards/app/discover#/?_g=(time:(from:'#{startTime | epoch_micros_to_date_iso}',to:'#{endTime | epoch_micros_to_date_iso}'))",
```
Comment (yurishkuro) [inline content/docs/next-release-v2/frontend-ui.md:72]: can we simplify this example? It does not need to be the exact Kibana query, and in the current form it's pretty difficult to read. Something like:
```suggestion
    "url": "my-logs.server/?from=#{startTime | add -60000000 | epoch_micros_to_date_iso}&to=#{endTime | add 60000000 | epoch_micros_to_date_iso}'",
```

**Review by yurishkuro — COMMENTED** [review]
let's combine with `add` function in one change

File: content/docs/next-release-v2/frontend-ui.md [hunk 3]
```diff
@@ -146,6 +151,17 @@ For traces, the supported template fields are: `duration`, `endTime`, `startTime
 
 Further, the trace template fields are available for substitution in process/logs/tags type when the trace template fields are prefixed with `trace.`. For example: `trace.traceID`, `trace.startTime`.
 
+#### Formatting
+
+In addition to interpolating fields into links, formatting functions can be used. The syntax is `#{field | function}` (eg `#{endTime | epoch_micros_to_date_iso}'`). The available formatting functions and a description of their behavior are:
+| Function name | Arguments | Description | Example |
```
Comment (yurishkuro) [inline content/docs/next-release-v2/frontend-ui.md:157]: Table results it very bad rendering where it bleeds into the RHS sidebar.

Let's use normal markup. It also makes function names into anchors.
```
##### pad_start
Pad the start of a string with a given character until the resulting string reaches the given length. Behavior is implemented by and thus matches...

Arguments:
  * `targetLength` (number) - ...
  * `padCharacter` (string) - ...
```
##### pad_start
Pad the start of a string with a given character until the resulting string reaches the given length. Behavior is implemented by and thus matches...

Arguments:
  * `targetLength` (number) - ...
  * `padCharacter` (string) - ...

**Review by yurishkuro — APPROVED** [review]
Thanks!

**Comment by drewcorlin1** [comment]
thanks for all the reviews @yurishkuro !

**Pull request merged**